test.sh

```diff
@@ -24,12 +30,92 @@ function prepareOutputDir() {
   mkdir -p output
 }
 
-function createTestEnvironment() {
+# Build DAP test container & start the cluster
+function createDAPTestEnvironment() {
+  # docker-compose build --pull client-dap cuke-master postgres test-dap
```

Comment:
Pull requests should not contain unused code, code commented out for testing, or temporary changes so this line and some of the other content needs uncommenting or removal completely.